modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -534,8 +534,8 @@ function editConsentStatusFields($db, $user)
  */
 function editCandidateDOD(\Database $db, \User $user): void
 {
-    $candID       = $_POST['candID'];
-    $dod          = $_POST['dod'];
+    $candID = $_POST['candID'];
+    $dod    = $_POST['dod'];
```

Comment:
It may be good to use `DateTime` here to validate whether this is a proper date string and throw an exception if it's not.